docs/update_tails_usbs.rst

```diff
@@ -0,0 +1,123 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you `back up your existing
+configuration`_ and upgrade as soon as possible.
+
+|Upgrade Notification|
+
+It usually takes some time for updates to download, so keep that in mind when choosing when to upgrade.
+
+.. |Upgrade Notification| image:: images/tails_upgrade_notification.png
+.. _back up your existing configuration: https://docs.securedrop.org/en/stable/backup_workstations.html
+
+Manual Updates
+--------------
+
+Because the *Secure Viewing Station* (SVS) is airgapped, it cannot
+receive upgrade notifications, so it will need to be updated manually.
+
+.. important:: 
+        The *SVS* stores some of SecureDrop's most sensitive data (decrypted submissions, the *Submission Private Key*), so we **strongly** encourage you to manually upgrade the *SVS* whenever a new version of Tails is released.
+
+The Tails project releases updates `every 6 to 8 weeks`_. Occasionally they
+release a new version ahead of the normal cycle in order to address a
+security issue. For regular Tails OS and security information, check out the
+Tails `Security page`_ and subscribe to the Tails RSS/Atom feed.
+
+.. _every 6 to 8 weeks: https://tails.boum.org/contribute/calendar/
+.. _Security page: https://tails.boum.org/security/index.en.html
+
+
+Tails Manual Update Process at a Glance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For simplicity, we'll refer to the Tails USB you want to upgrade as the *Secure Viewing Station* USB, although this process would work on any of your Tails USBs.
```

Comment:
re nit - sounds good, shall change to 'update' unless any objections (was trying to bridge the gap between our language and Tails' language).
re instructions: @eloquence and I went back and forth on this a little bit trying to make it sound clear (there's a lot of "the target USB, the USB you want to update, the primary USB that you update first" kind of thing, which I think is confusing). OTOH I want to mention the SVS because the whole point of this documentation was to make sure people keep the SVS up to date; on the other hand, it is true that it does work generically and that's important to capture (esp because when people get many versions behind on Tails, this is the easier upgrade path). I'll try to word it in a way that's clear.